src/containerapp/azext_containerapp/_help.py

```diff
@@ -2054,3 +2054,12 @@
           az containerapp job registry set -n my-containerapp-job -g MyResourceGroup \\
               --server MyContainerappJobRegistry.azurecr.io --identity system-environment
 """
+
+helps['containerapp debug'] = """
+    type: command
+    short-summary: Open an SSH-like interactive shell within a container app by debug console.
```

Comment:
```suggestion
    short-summary: Open an SSH-like interactive shell within a container app debug console.
```
better to keep aligned